addons/mail/static/src/messaging/entity/messaging/messaging.js

```diff
@@ -109,6 +98,10 @@ function MessagingFactory({ Entity }) {
          * Start messaging and related entities.
          */
         async start() {
+            this.update({
+                initializer: [['create']],
+                notificationHandler: [['create']],
+            });
```

Comment:
Same remark I did elsewhere, were I think the "link" should be done in "create" of the related entity (with adding "required" on those inverse fields in mind for later).

Reply:
Mmm... But all of those are one2one and they should be "required" both sides, so this is never going to work.

Reply:
`initializer` and `notificationHandler` are just split of concerns of `messaging`, they are currently implemented as separated entities, which is debatable because this introduces identity crisis and high coupling maintenance... Maybe we should invent something that looks like component hooks?